wsdl/ver10/events/wsdl/event.wsdl

```diff
@@ -80,11 +80,6 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 						<xs:documentation>Indicates that metadata streaming over MQTT is supported</xs:documentation>
 					</xs:annotation>
 				</xs:attribute>
-				<xs:attribute name="WSPullPointSupport" type="xs:boolean">
```

Comment:
As I recall, this was deprecated recently. Do we need to remove it immediately **without giving clients time to update their implementation**? Some old clients might still be referring to it (It will force the clients to update immediately to support new camera implementations). I suggest being cautious with this change since it is related to an event service which is very important.

Reply:
As there is an xs:anyAttribute in place old device implementation will not break clients. The only thing I can imagine is recompiling clients or devices may require them to remove code referencing the removed attribute which probably is a good thing.